bundle-size/app.js

```diff
@@ -211,7 +235,9 @@ module.exports = app => {
       head_sha: context.payload.pull_request.head.sha,
       output: {
         title: 'Calculating new bundle size for this PR…',
-        summary: 'Calculating new bundle size for this PR…',
+        summary: 'The bundle size (gzipped compressed size of `v0.js`) ' +
+          'of this pull request is being calculated on the ' +
+          '`INTEGRATION_TESTS` shard of the Travis build.',
```

Comment:
The name of the `integration_tests` shard is in lower case. 

Also, do we really want to point to a shard name? What if we decide to move things around later? (I'm fine leaving it in for now, since it might be helpful.)

Reply:
Yeah, I think it'll be helpful for developers to know where to look. Changed it to lower case. Done.